src/components/Input/Input.tsx

```diff
@@ -0,0 +1,124 @@
+import clsx from 'clsx';
+import type { ChangeEventHandler } from 'react';
+import React, { forwardRef } from 'react';
+import styles from './Input.module.css';
+
+export type InputProps = React.InputHTMLAttributes<HTMLInputElement> & {
```

Comment:
I think many of these attributes will be covered by unioning with `HTMLInputElement` already

Reply:
I removed `aria-describedby` prop but kept the others since they provide helpful hints 
When removed, there's no hint: ![Screenshot 2023-02-07 at 3 05 30 PM](https://user-images.githubusercontent.com/86632227/217387228-a30b11dd-eb3b-4db1-876f-1c2f9987467e.png)

Reply:
It's weird; i would have expected any internal docs to show for the attributes ... [but it doesn't seem to](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1848-L1915) 

I guess we can keep our own for now for documentation purposes